ingestion/src/metadata/ingestion/source/powerbi.py

```diff
@@ -100,7 +89,12 @@ def create(cls, config_dict, metadata_config: OpenMetadataServerConfig):
         Returns:
             PowerBiSource
         """
-        config = PowerbiSourceConfig.parse_obj(config_dict)
+        config = WorkflowSource.parse_obj(config_dict)
+        connection: PowerBIConnection = config.serviceConnection.__root__.config
+        if not isinstance(connection, PowerBIConnection):
+            raise InvalidSourceException(
+                f"Expected RedshiftConnection, but got {connection}"
```

Comment:
Use PowerBIConnection here pls 🙏 . It's hard to not miss this 😂  might need to find a better way to name this by passing the class or smt

Reply:
Resolved